cfme/fixtures/ansible_fixtures.py

```diff
@@ -165,8 +193,10 @@ def ansible_service_catalog(appliance, ansible_catalog_item, ansible_catalog):
 
 
 @pytest.fixture(scope="function")
-def ansible_service_request_funcscope(appliance, ansible_catalog_item):
-    request_descr = "Provisioning Service [{0}] from [{0}]".format(ansible_catalog_item.name)
+def ansible_service_request_funcscope(appliance, ansible_catalog_item_funcscope):
+    request_descr = "Provisioning Service [{0}] from [{0}]".format(
+        ansible_catalog_item_funcscope.name
+    )
```

Comment:
Since you're already formatting it, can you also add f-string instead of format.

Reply:
yes, will use f-strings